src/site/asn1.rst

```diff
@@ -44,6 +44,28 @@ Read the assignment description in its entirety before starting.
 Part 1 --- Country
 ==================
 
+You will be creating a class called ``Country`` that will hold some basic data. The data being stored will be the
+country ``name`` (string), ``population`` (int), ``area`` (double), and ``continent`` (string). This class will be used by the
+``CountryCatalogue`` class discussed in Part 2.
+
+1. Write a constructor for the ``Country`` class that takes four (4) parameters that will be stored as class fields.
```

Comment:
Definitely not cool for first years, but in case you ever write java in your work, I recommend [lombok](https://projectlombok.org/features/all) to write this garbage for you. `@Data` and `@Builder` and move on.

Literally this whole part is:
```java
@Data
class Country {
  String name;
  int population;
  double area;
  String continent;
}
```